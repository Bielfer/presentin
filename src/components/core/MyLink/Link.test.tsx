import { render } from '@testing-library/react';
import Link from '.';

describe('Link', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Link href="/">Test</Link>);

    expect(asFragment()).toMatchSnapshot();
  });
});
