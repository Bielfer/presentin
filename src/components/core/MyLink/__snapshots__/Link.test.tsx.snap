// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link matches snapshot 1`] = `
<DocumentFragment>
  <a
    class="cursor-pointer"
    href="/"
  >
    Test
  </a>
</DocumentFragment>
`;
